src/session.ts

```diff
@@ -9,8 +11,9 @@ export interface SessionStore<T> {
 }
 
 interface SessionOptions<S extends object> {
-  getSessionKey?: (ctx: Context) => Promise<string | undefined>
+  initial?: () => S
```

Comment:
Unused.

Reply:
I've removed `initial` for now. It can be added later if required. This PR is large enough.